### ISSUE TITLE
Difference behavior between FrozenBatchNorm2d and BatchNorm2d

### ISSUE THREAD

**Issue by b02202050** [pytorch/pytorch#43614]
## 🐛 Bug

The difference behavior of `torch.nn.BatchNorm2d` and `torchvision.ops.misc.FrozenBatchNorm2d` in eval mode.

## To Reproduce

```python
import torch
import torchvision
n1 = torch.nn.BatchNorm2d(1).eval()
n2 = torchvision.ops.misc.FrozenBatchNorm2d(1).eval()
n2.load_state_dict(n1.state_dict(), strict=False)
x = torch.rand(1,1,1,1)
torch.equal(n1(x), n2(x))
```
and I get a `False`

## Expected behavior

Their outputs should be the same.

## Environment

PyTorch version: 1.4.0a0+a5b4d78
Is debug build: False
CUDA used to build PyTorch: 10.2

OS: Ubuntu 18.04.3 LTS (x86_64)
GCC version: (Ubuntu 7.4.0-1ubuntu1~18.04.1) 7.4.0
Clang version: Could not collect
CMake version: version 3.14.0

Python version: 3.6 (64-bit runtime)
Is CUDA available: True
CUDA runtime version: 10.2.89
GPU models and configuration: 
GPU 0: Tesla V100-SXM2-32GB
GPU 1: Tesla V100-SXM2-32GB
GPU 2: Tesla V100-SXM2-32GB
GPU 3: Tesla V100-SXM2-32GB
GPU 4: Tesla V100-SXM2-32GB
GPU 5: Tesla V100-SXM2-32GB
GPU 6: Tesla V100-SXM2-32GB
GPU 7: Tesla V100-SXM2-32GB

Nvidia driver version: 440.33.01
cuDNN version: /usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.5

Versions of relevant libraries:
[pip] botorch==0.2.1
[pip] gpytorch==1.1.1
[pip] msgpack-numpy==0.4.3.2
[pip] numpy==1.17.4
[pip] pytorch-transformers==1.1.0
[pip] torch==1.4.0a0+a5b4d78
[pip] torchfile==0.1.0
[pip] torchnet==0.0.5.1
[pip] torchtext==0.4.0
[pip] torchvision==0.5.0a0
[conda] botorch                   0.2.1                    pypi_0    pypi
[conda] gpytorch                  1.1.1                    pypi_0    pypi
[conda] magma-cuda101             2.5.1                         1    local
[conda] mkl                       2019.1                      144  
[conda] mkl-include               2019.1                      144  
[conda] msgpack-numpy             0.4.3.2                  py36_0  
[conda] nomkl                     3.0                           0  
[conda] numpy                     1.17.4           py36hd5be1e1_0  
[conda] numpy-base                1.17.4           py36h2f8d375_0  
[conda] pytorch-transformers      1.1.0                    pypi_0    pypi
[conda] torch                     1.4.0a0+a5b4d78          pypi_0    pypi
[conda] torchfile                 0.1.0                    pypi_0    pypi
[conda] torchnet                  0.0.5.1                  pypi_0    pypi
[conda] torchtext                 0.4.0                    pypi_0    pypi
[conda] torchvision               0.5.0a0                  pypi_0    pypi

## Additional context

I found this problem when I am using the code from [this line](https://github.com/pytorch/vision/blob/fc69c22576cbccb59c581ddbaca4dedbdb279688/torchvision/models/detection/backbone_utils.py#L44)

**Comment by SsnL** [comment]
they have different eps https://github.com/pytorch/vision/blob/fc69c22576cbccb59c581ddbaca4dedbdb279688/torchvision/ops/misc.py#L54

**Comment by b02202050** [comment]
Thanks a lot!
